lib/spider/scripts/in_context_learning.py

```diff
@@ -32,21 +34,27 @@
 db_dir = DATASET_PATH / "database"
 table = DATASET_PATH / "tables.json"
 
+nltk.download("punkt")
+
 
 @hydra.main(version_base="1.3", config_path=str(CONFIG_PATH), config_name="eval.yaml")
 def main(cfg) -> None:
     transformers.set_seed(cfg.seed)
-
+    accelerator = Accelerator(log_with="wandb")
     # loggers
     setup_clearml(env_file_path=ENV_FILE_PATH)
     task = Task.init(
         project_name="spider",
         task_name="vanilla_llm",
         task_type=clearml.Task.TaskTypes.testing,
     )
-    run = wandb.init(project="spider")
-    if not isinstance(run, wandb_run.Run):
-        raise TypeError("Run is not a valid Wandb run")
+    accelerator.init_trackers(
+        project_name="spider",
+        config=OmegaConf.to_container(cfg),
+    )
+
+    # if not isinstance(run, wandb_run.Run):
+    # raise TypeError("Run is not a valid Wandb run")
```

Comment:
Either it's relevant code & you leave it like it is, or it's irrelevant one & you remove it (after all there we use a VCS tool so it's not lost!) 😉 
I think here you can just remove this as it's not relevant this you changed the way you log stuff